fulp_modules/jobs/deputy/code/deputy_zclothing.dm

```diff
@@ -0,0 +1,250 @@
+	icon = 'fulp_modules/jobs/deputy/deputy_clothing/ties_icons.dmi'
+	worn_icon =  'fulp_modules/jobs/deputy/deputy_clothing/ties_worn.dmi'
+	icon_state = "supply_tie"
+	inhand_icon_state = ""	//no inhands
+	w_class = WEIGHT_CLASS_SMALL
+	custom_price = PAYCHECK_EASY
+	var/department
+
+/obj/item/clothing/neck/fulptie/Initialize()
+	. = ..()
+	if(department)
+		name = "[department] tie"
+		desc = "A tie showing off that the user belongs to the [department] department."
+		icon_state = "[department]_tie"
+
+/obj/item/clothing/neck/fulptie/supply
+	department = "supply"
+
+/obj/item/clothing/neck/fulptie/engineering
+	department = "engineering"
+
+/obj/item/clothing/neck/fulptie/medical
+	department = "medical"
+
+/obj/item/clothing/neck/fulptie/science
+	department = "science"
+
+//Plasmamen clothes
+/obj/item/clothing/under/plasmaman/security/deputy
+	icon = 'fulp_modules/jobs/deputy/deputy_clothing/deputy_clothes.dmi'
+	worn_icon = 'fulp_modules/jobs/deputy/deputy_clothing/under_worn.dmi'
+	name = "deputy plasma envirosuit"
+	desc = "A plasmaman containment suit designed for deputies, offering a limited amount of extra protection."
+	icon_state = "deputy_envirosuit"
+	inhand_icon_state = "deputy_envirosuit"
+
+/obj/item/clothing/head/helmet/space/plasmaman/security/deputy
+	icon = 'fulp_modules/jobs/deputy/deputy_clothing/head_icons.dmi'
+	worn_icon = 'fulp_modules/jobs/deputy/deputy_clothing/head_worn.dmi'
+	name = "deputy envirosuit helmet"
+	desc = "A plasmaman containment helmet designed for deputies, protecting them from being flashed and burning alive, alongside other undesirables."
+	icon_state = "deputy_envirohelm"
+	inhand_icon_state = "deputy_envirohelm"
+
+//Berets
+/obj/item/clothing/head/fulpberet
+	armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 60, BIO = 0, RAD = 0, FIRE = 60, ACID = 60) // Same as Chaplain armor, it's the Deputy's holy beret.
+	worn_icon = 'fulp_modules/jobs/deputy/deputy_clothing/head_worn.dmi'
+	icon = 'fulp_modules/jobs/deputy/deputy_clothing/head_icons.dmi'
+
+/obj/item/clothing/head/fulpberet/supply
+	name = "supply deputy beret"
+	desc = "The headwear for only the most eagle-eyed Deputy, able to watch both Cargo and Mining."
+	icon_state = "beret_supply"
+
+/obj/item/clothing/head/fulpberet/engineering
+	name = "engineering deputy beret"
+	desc = "Perhaps the only thing standing between the supermatter and a station-wide explosive sabotage."
+	icon_state = "beret_engi"
+
+/obj/item/clothing/head/fulpberet/medical
+	name = "medical deputy beret"
+	desc = "This proud white-blue beret is a welcome sight when the greytide descends on chemistry."
+	icon_state = "beret_medbay"
+
+/obj/item/clothing/head/fulpberet/science
+	name = "science deputy beret"
+	desc = "This loud purple beret screams 'Dont mess with his matter manipulator!'"
+	icon_state = "beret_science"
+
+//Headsets
+/obj/item/encryptionkey/headset_depsupply
+	name = "supply deputy radio encryption key"
```

Comment:
do these need to exist at all though? security officers already have these, this is just bloat of stuff that already exists

Reply:
I thought it would be a cool thing if deputies wanted to get a second encryption key for more comms without being limited by removing sec or your department's comms

Reply:
¯\_(ツ)_/¯